src/models/model.ts

```diff
@@ -162,6 +174,69 @@ class Model {
             }
             return fontSize;
         });
+        getApi().onElectronKeyPress(this.onElectronKeyPress.bind(this));
+        this.keybindManager.registerKeybinding("electron", "any", (waveEvent) => {
```

Comment:
This is where we handle electron key events. The reason why we move this logic over here is so that we can use the stack inside of keybindManager